### PR TITLE
[ZH] Update Installation

### PR DESCRIPTION
**Pull request by Sendevia**

### CODE REVIEW

File: wiki/Installation/zh.md [hunk 1]
```diff
@@ -19,48 +19,48 @@ no_native_review: true
...
 3. 安装程序运行时，有10秒时间允许你选择安装位置。
    - 如果你想要修改安装位置，点击 `Hi!` 下方的文字来选择。
-   - 倒计时结束后，osu!将会被安装在这个位置里。
-4. 安装完成后，osu!将自动启动并使用osu!direct为你下载一些新手谱面。
-5. osu!会提示你登录或注册。
+   - 倒计时结束后，osu! 将会被安装在这个位置里。
+4. 安装完成后，osu! 将自动启动并使用 osu!direct 为你下载一些新手谱面。
+5. osu! 会提示你登录或注册。
    - 如果你已经有了账号，登录即可。
    - 如果你需要注册，参见[注册](/wiki/Registration).
```
Comment (Kotoki1337) [inline wiki/Installation/zh.md:26]:
```suggestion
   - 如果你需要注册，参见[注册](/wiki/Registration)。
```

Comment (Sendevia) [reply wiki/Installation/zh.md:26]: 感谢提醒😀

File: wiki/Installation/zh.md [hunk 2]
```diff
@@ -46,25 +52,26 @@ no_native_review: true
...
 
-1. 打开 osu!，然后点击主页右侧的`osu!direct`按钮。
+1. 打开 osu!，然后点击主页右侧的 `osu!direct` 按钮。
 2. 找一个你想玩的乐曲。
-3. 双击列表内的乐曲或点击右边的`Download`按钮来下载它。
-   - 如果你不想下载随谱面文件一起的视频文件，点击`DL NoVideo`。
+3. 双击列表内的乐曲或点击右边的 `Download` 按钮来下载它。
+   - 如果你不想下载随谱面文件一起的视频文件，点击 `DL NoVideo`。
 4. osu! 完成下载后，将会自动加载谱面集。
    - 如果你因为某些原因没看到它们，在歌曲选择页面内按下 `F5` 来重新处理谱面文件。
```
Comment (TPGPL) [inline wiki/Installation/zh.md:60]: song selection link missing

Comment (Sendevia) [reply wiki/Installation/zh.md]: ok, already added👌

**Pull request merged**